portfolio.html

```diff
@@ -20,46 +20,279 @@
-            <li>
-              <a href="#"
-                ><img
-                  src="images/social-media-icons/linkedin.png"
-                  alt="LinkedIn"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img src="images/social-media-icons/medium.png" alt="Medium"
-              /></a>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET’S CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/tw.png" alt="Twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="images/social-media-icons/linkedin.png"
+                    alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </section>
+        <section class="works">
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/tonic.png" alt="Tonic" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Tonic</h1>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="seprator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="seprator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default">See Project</button>
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/multi.png" alt="multi" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Multi-Post Stories</h1>
```

Comment:
- [ ] Kindly use headings in proper way , so you can use other headings multiple times in the document except `h1`